providers/amazon/tests/system/amazon/aws/example_appflow.py

```diff
@@ -29,6 +31,19 @@
 )
 from airflow.providers.standard.operators.bash import BashOperator
 
+if TYPE_CHECKING:
+    from airflow.models.baseoperator import chain
+    from airflow.models.dag import DAG
+else:
+    if packaging.version.parse(
```

Comment:
This should likely be extracted out to `version_compat.py` that you should place in the `system.amazon` pacjage and imported as

```
from system.amazon.version_compat import AIRFLOW_V_3_0_PLUS
```

This is the convention we used in a number of places.

Reply:
Also 2.10.0 here is not good. `task.sdk` can only be installed on Airflow 3+ and we already have 2.11 - so if `V3_0_PLUS` is a better condition for using `task.sdk`